### PR TITLE
Ht15

### PR DESCRIPTION
**Pull request by CherpakovPhilipp**

### CODE REVIEW

File: src/components/content/content.scss [hunk 1]
```diff
@@ -1,6 +1,6 @@
 @import '../../styles/variables.scss';
```
Comment (iliatcymbal) [inline src/components/content/content.scss]: you don't use any variable here

File: src/components/footer/footer.scss [hunk 2]
```diff
@@ -1,6 +1,6 @@
 @import '../../styles/variables.scss';
```
Comment (iliatcymbal) [inline src/components/footer/footer.scss]: here the same - you don't use any variable

File: src/components/header/header.scss [hunk 3]
```diff
@@ -1,6 +1,6 @@
 @import '../../styles/variables.scss';
```
Comment (iliatcymbal) [inline src/components/header/header.scss]: no need variables here

File: src/components/main/main.scss [hunk 4]
```diff
@@ -1,5 +1,5 @@
 @import '../../styles/variables.scss';
```
Comment (iliatcymbal) [inline src/components/main/main.scss]: why need variables?

File: package.json [hunk 5]
```diff
@@ -12,7 +12,7 @@
     "type": "git",
     "url": "git+https://github.com/iliatcymbal/test_markeet.git"
   },
-  "author": "iliatcymbmal",
+  "author": "CherpakovPhilipp",
```
Comment (iliatcymbal) [inline package.json]: :)
fix url to git repository as well

File: package.json [hunk 6]
```diff
@@ -12,7 +12,7 @@
...
   "license": "ISC",
   "bugs": {
     "url": "https://github.com/iliatcymbal/test_markeet/issues"
```
Comment (iliatcymbal) [inline package.json]: it is mine

File: src/components/footer/footer.scss [hunk 7]
```diff
@@ -1,7 +1,6 @@
-@import '../../styles/variables.scss';
 @import '../../styles/mixins.scss';
  
-.footer{
+footer{
```
Comment (iliatcymbal) [inline src/components/footer/footer.scss]: you can have few footers on the page, so it is better to use classes

File: package.json (hunk 5 above)
Comment (CherpakovPhilipp) [reply package.json]: oops))
Ok

**Pull request merged**